kubernetes.tf

```diff
@@ -17,6 +17,35 @@
 locals {
   # The Quortex cluster OIDC issuer.
   cluster_oidc_issuer = trimprefix(aws_eks_cluster.quortex.identity[0].oidc[0].issuer, "https://")
+
+  labels_tags = [
+    for key, node_group in var.node_groups :
+    {
+      for k, v in lookup(node_group, "labels", {}) :
+      key => {
+        "k8s.io/cluster-autoscaler/node-template/label/${k}" : v
+        "${k}" : v
+      }...
+    }
+  ]
+
+  asg_custom_tags_chunks = chunklist(flatten([
+    for key, node_group in var.node_groups : [
+      for k, v in merge({
+        nodegroup : key,
+        "k8s.io/cluster-autoscaler/node-template/label/nodegroup" : key
+        },
+        merge(local.labels_tags[0][key]...),
```

Comment:
I don't like doing that but I don't see anything better to reduce a list of map into a map.